test/e2e/testdata/ratelimit-global-shared-cidr-match.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: BackendTrafficPolicy
+metadata:
+  name: ratelimit-all-ips
+  namespace: gateway-conformance-infra
+spec:
+  targetRefs:
+    - group: gateway.networking.k8s.io
+      kind: HTTPRoute
+      name: cidr-ratelimit-1
+    - group: gateway.networking.k8s.io
+      kind: HTTPRoute
+      name: cidr-ratelimit-2
+  rateLimit:
+    type: Global
+    global:
+      shared: true
+      rules:
+        - clientSelectors:
+            - sourceCIDR:
+                value: 0.0.0.0/0
+                type: Distinct
+          limit:
+            requests: 3
+            unit: Hour
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: cidr-ratelimit-1
+  namespace: gateway-conformance-infra
+spec:
+  parentRefs:
+    - name: same-namespace
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /
```

Comment:
the routes look identical, can you update the path prefix for one to be `/foo` and other to be `/bar` and send requests to specific routes but check if the count/limit is shared ?